src/app/pages/vm/vm-edit-form/vm-edit-form.component.ts

```diff
@@ -132,31 +132,31 @@ export class VmEditFormComponent implements OnInit {
     const gpusIds = this.form.value.gpus;
 
     const pciIdsRequests$ = gpusIds.map((gpu) => {
-      return this.ws.call('device.get_pci_ids_for_gpu_isolation', [gpu]);
+      return this.ws.call('vm.device.get_pci_ids_for_gpu_isolation', [gpu]);
     });
 
-    forkJoin(pciIdsRequests$).pipe(
+    const updateVm$: Observable<unknown> = pciIdsRequests$.length ? forkJoin(pciIdsRequests$).pipe(
```

Comment:
I think we can make this simpler by using variables with readable names and if/else instead of multi-line conditionals.